src/lib/loader.js

```diff
@@ -134,24 +124,18 @@ export async function swapContent(isFirstRun) {
   ga("send", "pageview");
 
   // Remove the current #content element
-  main.querySelector("#content").remove();
-  main.appendChild(page.querySelector("#content"));
+  main.querySelector("#content").innerHTML = partial.raw;
 
   // Update the page title
-  document.title = page.title;
-  // Update the page description
-  const description = page.querySelector("meta[name=description]");
-  const updatedContent = description ? description.content : "";
-  document.querySelector("meta[name=description]").content = updatedContent;
+  document.title = partial.title || "";
 
   // Focus on the first title (or fallback to content itself)
   forceFocus(content.querySelector("h1, h2, h3, h4, h5, h6") || content);
 
-  // Determine if this was the offline page
-  const isOffline = Boolean(getMeta("offline", page));
-
+  // Update state including whether the Service Worker actually returned the
+  // offline page instead of our actual content
   store.setState({
     isPageLoading: false,
```

Comment:
Good point. Resolved. Having said that, I'm not really sure doing it in the `finally` block matters, since we don't allow the code to continue after this _anyway_. I've removed that.